oeps/best-practices/oep-0011/decisions/0006-codecov.rst

```diff
@@ -0,0 +1,29 @@
+Use Codecov to measure code covered by tests
+############################################
+
+Status
+******
+
+Accepted
+
+Context
+*******
+
+It is important for the health of the Open edX project to measure the amount of code covered by automated testing.
+
+Decision
+********
+
+For measuring test coverage, `Codecov`_ should be used.
+
+Consequence
+***********
+
+`Codecov`_ has been adopted by the Open edX community as the tool for measuring test coverage.
+
+Rejected Alternatives
+*********************
+
+At the time of this decision, no rejected alternatives were documented.
+
```

Comment:
```suggestion
Coveralls
=========

Coveralls has been historically used for tracking coverage but the service has become less reliable over time and the larger python has moved away from it. 

```

Some historical context that I remember.

Reply:
Thank you! I added it via fixup https://github.com/openedx/open-edx-proposals/compare/9a4aff3f803e2353522972809ddb9659d7f70cf7..98c1bbf5bcc4697c8d1310874b94802778957b54